src/main/scala/viper/gobra/translator/encodings/structs/StructEncoding.scala

```diff
@@ -243,22 +243,22 @@ class StructEncoding extends TypeEncoding {
 
   /**
     * Generates:
-    * function shStructDefault(): [Struct{F}@]
-    *   ensures AND (f: T) in F. [&result.f == dflt(T)]
+    * function shStructDefault(): [res: Struct{F}@]
+    *   ensures AND (f: T) in F. [&res.f == dflt(T)]
     */
   private val shDfltFunc: FunctionGenerator[Vector[in.Field]] = new FunctionGenerator[Vector[in.Field]] {
     override def genFunction(fs: Vector[in.Field])(ctx: Context): vpr.Function = {
-      val resType = in.StructT("the name does not matter", fs, Shared)
+      val resType = in.StructT(fs, Shared)
       val vResType = typ(ctx)(resType)
       val src = in.DfltVal(resType)(Source.Parser.Internal)
-      val resDummy = in.LocalVar(Names.freshName, resType)(src.info)
+      val resDummy = in.LocalVar("res", resType)(src.info)
```

Comment:
Is the justification that using "res" is fine because there are no other variables? Please, write this as a comment.